### ISSUE TITLE
In the doc deploy by docker image with digest's sha example is wrong for ghcr.io

### ISSUE THREAD

**Issue by olup** [dokku/dokku#4963]
It's a small thing but took me some time to figure out why I was getting an `invalid reference format`.

For me, pulling an image from ghcr.io by sha, the example would not work : 

```
dokku git:from-image node-js-app dokku/node-js-getting-started:@sha256:9d187c3025d03c033dcc71e3a284fee53be88cc4c0356a19242758bc80cab673
```
I had to remove the colon before  `@sha`. This would work:
```
dokku git:from-image node-js-app dokku/node-js-getting-started@sha256:9d187c3025d03c033dcc71e3a284fee53be88cc4c0356a19242758bc80cab673
```

**Comment by josegonzalez** [comment]
Mind making a pull request to fix the example?

**Comment by olup** [comment]
Sure, I'll try to do that tomorrow - it's pretty late where I am.

**Comment by josegonzalez** [comment]
Closed by #4992.
